07_safety/01_agency_geo_county.Rmd

```diff
@@ -0,0 +1,731 @@
+
+To learn more about NIBRS data, consult Jacob Kaplan's book titled [National Incident-Based Reporting System (NIBRS) Data: A Practitioner's Guide](https://nibrsbook.com/). For these metrics, the most important thing to know about these data is that most police agencies do not report NIBRS data prior to 2021. In 2019, only 8,500 out of approximately 18,000 police agencies in the United States (covering about 45% of the US population) reported NIBRS data. Instead, agencies primarily reported [UCR data](https://ucrbook.com/) for 2020 and before. However, the FBI has moved entirely to NIBRS data starting in 2021 and no longer collects UCR data, which is why we use NIBRS data instead ([source](https://nibrsbook.com/overview-of-the-data.html)). 
+
+We do not want to use NIBRS data for some years and UCR data for others. However, because of lack of reporting, **there are a lot of missing values prior to 2021 for NIBRS data**. Future updates of the safety metrics may want to include historical NIBRS data back to 2014 and should keep this in mind.
+
+    Note that we only use the 2022 Batch Header File. Using the Batch Header File from each year might be more accurate, but for the most part it won't matter, especially because so few agencies reported through NIBRS prior to 2021. Additionally, downloading and reading in the Batch Header File for each year takes a lot of time and computing power, and must be done manually for each year which introduces room for error in replication. Future updates may consider using the Batch Header Files for each year, which can be accessed through the following links:
+    * [2014](https://www.icpsr.umich.edu/web/ICPSR/studies/36421)
+    * [2015](https://www.icpsr.umich.edu/web/ICPSR/studies/36851)
+    * [2016](https://www.icpsr.umich.edu/web/ICPSR/studies/37066)
+    * [2017](https://www.icpsr.umich.edu/web/ICPSR/studies/37650)
+    * [2018](https://www.icpsr.umich.edu/web/ICPSR/studies/37649)
+    * [2019](https://www.icpsr.umich.edu/web/ICPSR/studies/38565) 
+    * [2020](https://www.icpsr.umich.edu/web/ICPSR/studies/38566) 
+    * [2021](https://www.icpsr.umich.edu/web/ICPSR/studies/38807)
+    * [2022](https://www.icpsr.umich.edu/web/ICPSR/studies/38925)
+    
+Accessing these data requires an ICPSR account. Manually downloading these data is a five-step process, but you will only need to do each of these steps once for the creation of this metric:
+    1. Create an ICPSR account [here](https://login.icpsr.umich.edu/realms/icpsr/login-actions/registration?client_id=icpsr-web-prod&tab_id=BqotoE6wnqw).
```

Comment:
Note that this lead me to a dead link (or said that I need to enable cookies) this might be a better link? https://www.icpsr.umich.edu/cgi-bin/newacct

Reply:
Thank you! Not sure why the other link stopped working, but I have updated it with the link you found.